images/win/scripts/ImageHelpers/InstallHelpers.ps1

```diff
@@ -386,3 +386,24 @@ function Test-IsWin16
 {
     (Get-WinVersion) -match "2016"
 }
+
+function Extract-7Zip {
+    param
+    (
+        [Parameter(Mandatory=$true)]
+        [string]$Path,
+        [Parameter(Mandatory=$true)]
+        [string]$DestinationPath
+    )
+
+    try{
```

Comment:
try-catch doesn't work for external commands, use $LASTEXITCODE variable and check exit code.